Makefile

```diff
@@ -297,9 +297,9 @@ failpoint-disable: tools/bin/failpoint-ctl
 	@$(FAILPOINT_DISABLE)
 
 .PHONY: tools/bin/ut
-tools/bin/ut: tools/check/ut.go
+tools/bin/ut: tools/check/ut.go tools/check/longtests.go
 	cd tools/check; \
-	$(GO) build -o ../bin/ut ut.go
+	$(GO) build -o ../bin/ut ut.go longtests.go
```

Comment:
Clould we also support `make ut-long`?

Reply:
Yeah. It'll be a good QoL change. Let me do it.

Reply:
Done.